locales/en/kafkaoverview.json

```diff
@@ -10,5 +10,16 @@
   "purchaseCardCallToActionButton": "Purchase now",
   "purchaseCardMainText": "Use Red Hat Marketplace to purchase a pay-as-you-go subscription for up to 25 streaming units. Purchasing through Red Hat Marketplace is only available in the United States and Canada.",
   "purchaseCardTitle": "Purchase now",
-  "viewDocumentation": ""
+  "viewDocumentation": "",
+  "pricingModalTitle": "Pricing model",
+  "steamingUnit": "Streaming Unit",
+  "dataTransfer": "Data Transfer",
+  "storage": "Storage",
+  "kafkaInstanceCapacityTitle": "Kafka instance capacity",
+  "kafkaInstanceCapacityDescription": "OpenShift Streams offers instances in the following sizes with the following capacity",
+  "kafkaInstanceCapacityfooter": "*Storage can be extended to up to 54 TB. If you wish to increase the storage limit for a Kafka instance, contact <support>Red Hat Support</support> to discuss your options. You can check your current storage usage by <value>monitoring disk space metrics</value>.",
```

Comment:
After checking the [limits document](https://docs.google.com/spreadsheets/d/1W4H1IgDXKv24Sf7aWeqeAqxjKVj8EJB9giYN9obqPgU/edit#gid=1963544753), the design doesn't support showing separate limits for each size, and the size shown here (54 TB) isn't included at all in that document.

For now, let's change this text to this:
```
"kafkaInstanceCapacityfooter": "*Storage can be extended. If you wish to increase the storage limit for a Kafka instance, contact <support>Red Hat Support</support> to discuss your options. You can check your current storage usage by <value>monitoring disk space metrics</value>.",
```